### DISCUSSION TITLE
Tester here!

### DISCUSSION THREAD

**Discussion by jacojacker** [infinitymdm/heroic-games-solus#2] [General]
Hi! I've tried to package it before but as I don't have the knowledge I might help as a tester.

My system is an AMD X64 with a RX  6600 XT gpu (all open source drivers).

As far as the installation goes, I had no  problem at all.

Neither by using the application.

PS : Not sure if it is the right place to post this test.

If you need any other info, just let me know.

Cheers!

**Comment by infinitymdm** [comment]
@jacojacker I appreciate you taking the time to test this out! I'm working on getting it pulled into the official repository, so hopefully it will be available directly via eopkg soon.

For the time being, I've also opened discussions up so that positive test reports don't show up as issues. Future testing should go there. 

If you do have issues with installation (say, dependency trouble or something) in the future, of course open an issue and I'll take a look. Note that issues with the Heroic app itself should be [submitted to Heroic](https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/issues).  

Thanks for testing!

**Comment by jacojacker** [comment]
Hi @infinitymdm.

Thanks for the fast packaging after the update from Heroic!

As soon as I've download it, tried installing with the 'eopkg it' command, and I got this as result :

`sudo eopkg it --debug heroic-games-launcher-2.2.2-2-1-x86_64_unstable.eopkg
DEBUG: HistoryDB initialized in 0.000711917877197.
DEBUG: A = ['heroic-games-launcher-2.2.2-2-1-x86_64_unstable.eopkg']
DEBUG: InstallDB initialized in 0.00240015983582.
DEBUG: PackageDB initialized in 0.0298960208893.
DEBUG: RepoDB initialized in 0.000136852264404.
Error del sistema. Programa terminado.
<type 'exceptions.Exception'>: Dependencias externas no satisfechas: expat release >= 24
Favor use  'eopkg help' para ayuda general.

Trazo inverso:
  File "/usr/bin/eopkg", line 82, in <module>
    cli.run_command()
  File "/usr/lib/python2.7/site-packages/pisi/cli/pisicli.py", line 142, in run_command
    self.command.run()
  File "/usr/lib/python2.7/site-packages/pisi/cli/install.py", line 104, in run
    pisi.api.install(packages, ctx.get_option('reinstall') or reinstall)
  File "/usr/lib/python2.7/site-packages/pisi/api.py", line 69, in wrapper
    ret = func(*__args,**__kw)
  File "/usr/lib/python2.7/site-packages/pisi/api.py", line 471, in install
    return pisi.operations.install.install_pkg_files(packages, reinstall)
  File "/usr/lib/python2.7/site-packages/pisi/operations/install.py", line 201, in install_pkg_files
    raise Exception(_('External dependencies not satisfied: %s') % dep)
`
It seems that my systems lacks the expat binary for this update, probably because I am on the main Solus repository and not the experimental?

Regards!

**Reply by infinitymdm** [reply]
That's exactly correct. For the time being I'd suggest sticking with the 2.2.1 release. This update doesn't add a whole lot if you're only using Epic, as most of the fixes were for GOG.

**Reply by jacojacker** [reply]
Thanks! Will do that

PS : Next time I'll post it where it belongs, sorry!

**Reply by infinitymdm** [reply]
A quick update on this: the folks over at Solus are reviewing my changes for 2.2.2 and realized that most of the dependencies aren't actually required. I'll publish a new release this afternoon (probably 2.2.2a or something) that will remove the unneeded dependencies. You'll probably be able to install that one without issue. I'll let you know when it's posted.

**Reply by infinitymdm** [reply]
Just posted the new release. Let me know if you have any trouble. Enjoy!
